bukkit/src/main/resources/plugin.yml

```diff
@@ -6,7 +6,7 @@ description: ${description}
 api-version: 1.13
 folia-supported: true
 loadbefore: [ ProtocolLib, ProxyPipe, SpigotLib, SkinRestorer ]
```

Comment:
"SkinsRestorer" is spelled wrong in `loadbefore`.

Reply:
I believe it wasn't when this got added, but seems like they changed it to SkinsRestorer